Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
        Proxy = ProgressBarManager.Proxy
        queue = MockQueue()
        p = Proxy(queue)
    
        def assert_update(n):
            assert n == 1
    
        p.update = assert_update
>       list(p.new([1,2,3,4]))

../test.py:22: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
flutes/multiproc.py:933: in _iter_per_elems
    self.update(idx - prev_index)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

n = 2

    def assert_update(n):
>       assert n == 1
E       assert 2 == 1

../test.py:19: AssertionError
=========================== short test summary info ============================
FAILED ../test.py::test - assert 2 == 1
============================== 1 failed in 0.10s ===============================
